src/Select.js

```diff
@@ -1107,9 +1126,9 @@ class Select extends React.Component {
 					</span>
 					{removeMessage}
 					{this.renderLoading()}
-					{this.renderClear()}
 					{this.renderArrow()}
 				</div>
+				{this.props.multi && (!isOpen && this.renderBottomClear())}
```

Comment:
Probably the `this.renderClear()` should stay, because the new `Clear All` will be only on `multi`, so if there is a select with a single option that should display the remove button it will not work, so it may be better to have a retrocompatibility, like:
```javascript
{!this.props.multi && this.renderClear()}
```